src/fs/file.rs

```diff
@@ -204,6 +204,7 @@ impl<'dir> File<'dir> {
     /// Returns an IO error upon failure, but this shouldn’t be used to check
     /// if a `File` is a directory or not! For that, just use `is_directory()`.
     pub fn to_dir(&self) -> io::Result<Dir> {
+        trace!("to_dir reading dir");
```

Comment:
Just a small display change
```suggestion
        trace!("to_dir: reading dir");
```